### DISCUSSION TITLE
How to set the flyout background transparent

### DISCUSSION THREAD

**Discussion by jeff9571** [AvaloniaUI/Avalonia#12291] [Q&A]
I follow the documentation to style the flyout, the background can be set to other colors, but not transparent. 
Here is the style:
`<Style Selector="FlyoutPresenter.DefaultFlyout">
            <Setter Property="Background" Value="Transparent" />
            <Setter Property="Padding" Value="0" />
        </Style>`
Here is the button's flyout
` <Button.Flyout>
                                <Flyout FlyoutPresenterClasses="DefaultFlyout" Placement="Bottom" ShowMode="Standard">
                                    <StackPanel>
                                        <Button  Content="Test" HorizontalAlignment="Stretch"></Button>
                                    </StackPanel>
                                </Flyout>
                            </Button.Flyout>`

![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/4751756f-e2cf-42dc-b6a6-4730c786f30f)

**Comment by timunie** [comment]
works for me using Windows 11 and Avalonia 11.0. I have changed Window Background to make it more visible: 

![image](https://github.com/AvaloniaUI/Avalonia/assets/47110241/32a08acd-8e38-4651-a6bd-d39fd9c7ff29)
